tests/Hydrators/PropertyHydratorTest.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+require('vendor/autoload.php');
+
+use ArmadilloEstates\Hydrators\PropertyHydrator;
+
+class PropertyHydratorTest extends PHPUnit\Framework\TestCase
+{
+    public function testSuccessHydrator()
+    {
+        $pdoMock = $this->createMock(PDO::class);
+        $hydrator = new PropertyHydrator($pdoMock);
+        $this->assertObjectHasAttribute('database', $hydrator);
```

Comment:
dont need to do this, the object canot not have that attribute if it has been created correctly, just use:
https://phpunit.readthedocs.io/en/9.0/assertions.html#assertinstanceof